### PR TITLE
bugfix/13884-scrollablePlotArea-collaps-nodes

### LINKED ISSUE #13884: Gantt : Collapsable nodes + navigator +  chart.scrollablePlotArea
#### Expected behaviour
I want to fix header position (with chart.scrollablePlotArea), 
using navigator option and collapsable nodes. 

#### Actual behaviour
When i add scrollableplotarea to my charts, i have several bugs : 
 - navigator is mostly not displayed, and i have a white bloc in the page bottom
 - when i close nodes (all nodes for example), all the chart disappear 
 - opactiy of the header is OK on the first display, then when we close nodes, opacity becomes to 0

#### Live demo with steps to reproduce
template: https://jsfiddle.net/yfojxqb4/4/ 

#### Product version
Highcharts.ganttChart 8.1.2

#### Affected browser(s)
I am using Chrome latest version.

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #13884, collapsing node with `scrollablePlotArea` crashed the chart.

### CODE REVIEW

File: ts/Extensions/StaticScale.ts [hunk 1]
```diff
@@ -98,10 +98,12 @@ Chart.prototype.adjustHeight = function (): void {
 
                 diff = height - chart.plotHeight;
 
-                if (Math.abs(diff) >= 1) {
-                    chart.plotHeight = height;
-                    chart.redrawTrigger = 'adjustHeight';
-                    chart.setSize(void 0, chart.chartHeight + diff, animate);
+                if (!chart.scrollablePixelsX && !chart.scrollablePixelsY) {
```
Comment (TorsteinHonsi) [inline ts/Extensions/StaticScale.ts:101]: Do we need to check both the X and the Y direction? Isn't it enough to block static scale only in the vertical direction?

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! In addition to the inline comment, do we have a good explanation for the difference in the visual test?

File: ts/Extensions/StaticScale.ts (hunk 1 above)
Comment (karolkolodziej) [reply ts/Extensions/StaticScale.ts]: You are right! There is no need to check both.

**Comment by karolkolodziej** [comment]
I went through the demo from visual differences, and it's hard to find the origin of that difference.
When you review the file difference, you can see that the width of the chart was changed.
Inside this PR, I didn't play with the chart width, so I guess that these changes are irrelevant. Correct me please if I'm wrong.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks! Approved, awaiting CI tests after the last commit.

**Comment by TorsteinHonsi** [comment]
The actual fix in the PR (checking `scrollablePixelsY`) is already applied in the master. So if the original issue still persists, it must be caused by something else.

**Pull request closed without merging**